gui/options.cpp

```diff
@@ -2069,13 +2070,26 @@ void GlobalOptionsDialog::addMiscControls(GuiObject *boss, const Common::String
 		_autosavePeriodPopUp->appendEntry(_(savePeriodLabels[i]), savePeriodValues[i]);
 	}
 
+	_guiReturnToLauncherAtExit = new CheckboxWidget(boss, prefix + "ReturnToLauncherAtExit",
```

Comment:
Please add a check on the `kFeatureNoQuit` here. If it is on, this option makes no sense so we should probably not add it to the GUI. I though about it, but forgot to comment during my initial review.

Reply:
Ah, right - I absolutely missed that too. Thanks, fix incoming!